pbgzip/Makefile

```diff
@@ -1,7 +1,7 @@
 CC=			gcc
 CFLAGS=		-g -Wall -O2 -pthread
 #LDFLAGS=		-Wl,-rpath,\$$ORIGIN/../lib
-DFLAGS=		-D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -DHAVE_LIBPTHREAD #-DDISABLE_BZ2
+DFLAGS=		-D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -DHAVE_LIBPTHREAD -DDISABLE_BZ2
```

Comment:
I am on vacation for a few more days hence the zero activity. Please be patient. :)

Thumb typed for added typos

> On Aug 25, 2015, at 7:13 AM, Guillermo Carrasco <notifications@github.com> wrote:
> 
> In pbgzip/Makefile:
> 
> > @@ -1,7 +1,7 @@
> >  CC=			gcc
> >  CFLAGS=		-g -Wall -O2 -pthread
> >  #LDFLAGS=		-Wl,-rpath,\$$ORIGIN/../lib
> > -DFLAGS=		-D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -DHAVE_LIBPTHREAD #-DDISABLE_BZ2
> > +DFLAGS=		-D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_USE_KNETFILE -DHAVE_LIBPTHREAD -DDISABLE_BZ2
> Hi @nh13,
> 
> I also experienced that if I do uncomment this flag and also remove -lbz2 form below, I can compile, otherwise I get a compiling error:
> 
> gcc -g -Wall -O2 -pthread -o pbgzip ../bgzf.o ../knetfile.o util.o block.o consumer.o queue.o reader.o writer.o pbgzf.o pbgzip.o   -lm -lz -lbz2
> consumer.o: In function `consumer_run':
> consumer.c:(.text+0x3f3): undefined reference to `consumer_init_header'
> consumer.c:(.text+0x47c): undefined reference to `consumer_update_header'
> collect2: error: ld returned 1 exit status
> I´m trying on a CentOS 6.6 x64. Also on a MacOS where I get a different error, but maybe start first with linux :)
> 
> Do you think you could have a look at this? We would be happy to help you fork out this useful tool into a separate repo.
> 
> Thanks!
> 
> —
> Reply to this email directly or view it on GitHub.
>